wemake_python_styleguide/violations/consistency.py

```diff
@@ -2102,3 +2104,33 @@ class IterableUnpackingViolation(ASTViolation):
 
     error_template = 'Found an unnecessary iterable unpacking'
     code = 356
+
+
+@final
+class LineCompriseCarriageReturnViolation(TokenizeViolation):
+    """
+    Forbids to use \r (carriage return) in line breaks.
+
+    Reasoning:
+        We enforce strict consitency rules about how to break lines in 'str' type.
```

Comment:
Should be: `We enforce Unix-style newlines.` That's it.

Reply:
Ok, I'll change describing

Reply:
fixed